ports/ereignis/vcpkg.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "ereignis",
+  "version": "1.1",
+  "description": "A thread-safe C++17 Event Library",
+  "homepage": "https://github.com/Soundux/ereignis",
+  "license": "MIT",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
```

```suggestion
    }
```
`vcpkg-cmake-config` is not used.